python-avd/pyavd/_eos_designs/structured_config/network_services/router_ospf.py

```diff
@@ -76,7 +76,7 @@ def router_ospf(self: AvdStructuredConfigNetworkServices) -> dict | None:
                     if (route_map := get(vrf, "ospf.redistribute_bgp.route_map")) is not None:
                         process_redistribute["bgp"]["route_map"] = route_map
 
-                if get(vrf, "ospf.redistribute_connected.enabled", default=False) is True:
+                if get(vrf, "redistribute_connected", get(vrf, "ospf.redistribute_connected.enabled", default=False)) is True:
```

Comment:
Suggesting to revert this change so we keep BGP and OSPF logic separate.
```suggestion
                if get(vrf, "ospf.redistribute_connected.enabled", default=False) is True:
```

Reply:
FIxed.